.travis.yml

```diff
@@ -16,3 +16,4 @@ env:
     global:
         - FACEBOOK_APP_ID=198798870326423
         - FACEBOOK_SECRET=2db4d76fe8a336cf292470c20a5a5684
+        - FACEBOOK_USER_ID=12345
```

Comment:
@juhopaak added methods to create test users programmatically (fdc36ccd330f98c20ea6a0b001687d815b17ff97). I think using them is a better solution than using a hardcoded user ID.